File: python/selectionSort.py
def selectionSort(mylist):
    for i in range(len(mylist)-1):
        minIdx = i
        for j in range(i+1, len(mylist)-1):
            if mylist[j] < mylist[minIdx]:
                minIdx = j
        temp = mylist[minIdx]
        mylist[minIdx] = mylist[i]
        mylist[i] = temp

    return mylist

File: java/SelectionSort.java
import java.util.Arrays;

public class SelectionSort {
    static int[] nums;

    public static void main(String[] args) {
        nums = new int[10];
        for (int i=0;i<10;i++) {
            nums[i] = (int) (Math.random() * 10);
        }

        System.out.println("<정렬전>");
        System.out.println(Arrays.toString(nums));

        for (int i=0;i<nums.length-1;i++) {
            int minIndex = i;
            for (int j=i+1;j<nums.length;j++) {
                if (nums[minIndex] > nums[j]) {
                    minIndex = j;
                }

                int temp = nums[minIndex];
                nums[minIndex] = nums[i];
                nums[i] = temp;
            }
        }

        System.out.println("<정렬후>");
        System.out.println(Arrays.toString(nums));
    }
}
